sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -511,33 +529,32 @@ def _from_generated(cls, generated):
 
 
 class FilteredBlob(DictMixin):
-    """Blob info from a Filter Blobs API call.
-
-    :ivar name: Blob name
-    :type name: str
-    :ivar container_name: Container name.
-    :type container_name: str
-    :ivar tags: Key value pairs of blob tags.
-    :type tags: Dict[str, str]
-    """
-    def __init__(self, **kwargs):
+    """Blob info from a Filter Blobs API call."""
+
+    name: Optional[str] 
```

Comment:
Probably the same case with optional here.

Let's re-evaluate all the public facing models here.